src/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/e4fb871c71.js" crossorigin="anonymous"></script>
+    <script src="https://unpkg.com/lodash@4.17.20"></script>
+    <link href="./style.css" type="text/css" rel="stylesheet"/>
+    <title>TO_DO_LIST</title>
+</head>
+<body>
+    <div class="the-list">
+
+        <div class="list-container">
+           <div class="today-to">
+              <p id="today-title">Today's To Do</p>
+              <img id="sync" src="sync.png" alt="sync">  
+              <hr>
+           </div>
+
+        <div class="the-input">
+            <input type="text" id="theinput"  placeholder="Add to your list...">
+            <img id="enter" src="enter.png" alt="enter">
+            <hr>
+        </div>
+          <ul class="to-dos">    </ul>
+       </div>
+
+      <div class="clear">
+        <a href="#"><p id="clear-p">Clear all completed</p></a>
+      </div>
+
+    </div>
+
+    <img class="ellipsis" src="ellipisis.png" alt="ellipisis">
+    <img class="del" src="delete.png" alt="del">
+
```

Comment:
- [ ] Kindly, remove these images from your UI to make it clean and user-friendly. 
![image](https://user-images.githubusercontent.com/71874425/184455026-1ec8d8c5-7307-4356-96e9-b5a755524eec.png)

Reply:
removed

Reply:
DONE